src/pyhf/tensor/pytorch_backend.py

```diff
@@ -322,8 +324,11 @@ def normal_cdf(self, x, mu=0.0, sigma=1.0):
         Returns:
             PyTorch FloatTensor: The CDF
         """
-        normal = torch.distributions.Normal(mu, sigma)
-        return normal.cdf(x)
+        # the following variant is numerically more stable for low p-values
+        # than the default version that uses torch.erf
+        # erfc(x) = 1 - erf(x)
```

Comment:
Can there be a plot that is added to the PR similar to how was done in Issue #901 that demonstrates the improved stability?

Reply:
i added a comment below

Reply:
Thanks! It is exactly what I was hoping for. :)